genai/client.go

```diff
@@ -213,6 +213,19 @@ func (m *GenerativeModel) newCountTokensRequest(contents ...*Content) *pb.CountT
 	}
 }
 
+func (m *GenerativeModel) Info(ctx context.Context) (*ModelInfo, error) {
```

Comment:
Needs a docstring here and in embed.go

Please tag a new release when this is merged

Reply:
Done.